docs/relational-databases/performance-monitor/use-sql-server-objects.md

```diff
@@ -38,12 +38,12 @@ ms.author: jrasnick
   
  Some objects have several instances if multiple resources of a given type exist on the computer. For example, the **Processor** object type will have multiple instances if a system has multiple processors. The **Databases** object type has one instance for each database on [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]. Some object types (for example, the **Memory Manager** object) have only one instance. If an object type has multiple instances, you can add counters to track statistics for each instance, or in many cases, all instances at once. Counters for the default instance appear in the format **SQLServer:**_\<object name>_. Counters for named instances appear in the format **MSSQL$**_\<instance name>_**:**_\<counter name>_ or **SQLAgent$**_\<instance name>_**:**_\<counter name>_.  
   
+ SQL Server performance counters values uses the Windows Performance Counter engine. Some values are not calculated directly by SQL Server engine. SQL Server just provide base values to performance counter engine and it will do the work for calculation (like show percentages, etc.). The [sys.dm_os_performance_counters &#40;Transact-SQL&#41;](../../relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql.md) dynamic management view provide all counters with original value generated by SQL Server, and the column `cntr_type` indicates the type of counter. How performance counter engine will process this value depend on this type. For more detailed about these types, [check WMI documentation about performance counters](https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-performance-counter-types)

```

Comment:
Please replace SQL Server with the token `[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]`, and SQL Server engine with `[!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]`. 

Suggested redaction:

```
[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] performance counters values are generated using the Windows Performance Counter (WPC) engine. Some counter values are not calculated directly by [!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]. The [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] provides base values to the WPC engine which will perform the required calculations (like percentages). The [sys.dm_os_performance_counters &#40;Transact-SQL&#41;](../../relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql.md) dynamic management view provides all counters with the original value generated by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], and the column `cntr_type` indicates the type of counter. How the WPC engine processes [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] counter values depends on this type. For more details about performance counter types, see the [WMI documentation](https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-performance-counter-types).
```

Reply:
Your suggestions was applied. I didn't know that we must use this, and this is nice. I will pay more attention on next submissions. Thanks.